src/main/java/com/example/Restaurant_Management_System_REST_API/DTO/InventoryItemDTOs/InventoryItemDTOResponse.java

```diff
@@ -21,7 +22,7 @@ public InventoryItemDTOResponse(Long id, int amount, Supplier supplier,
 
     private Long id;
     private int amount;
-    private Supplier supplier; //TODO: change to DTO
```

Comment:
Why it's not changed into DTO then? Conception changed here?

Reply:
exactly! SupplierDTO is the same as Supplier so I decided to stay as it is

Reply:
Okay, I see. I'd add a DTO anyway, so we don't expose our model to the outside world. That's the first point.

Second would be about expadability of the project. You decide if this solution satisfies you atm.

Reply:
this little change in code would bring dozens of changes. I tried to do this and this is like domino. I don't feel this is worth to do right now. It demands a lot of work and I don't see an add value at this moment. Furthermore this could also lead to malfunction of the app.

Reply:
I'm refactoring right now!

Reply:
okay! after the struggle I think it is much better now!